src/components/AboutSection/AboutSection.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import classNames from "classnames/bind";
+import AboutImg from "./AboutImg/AboutImg";
+import AboutTextContainer from "./AboutTextContainer";
+
+const AboutSection = ({ data, src, srcSet }) => (
+    <section className={classNames("container about", data.sectionClass)}>
+        <AboutTextContainer data={data} />
+        <AboutImg src={src} srcSet={srcSet} />
+    </section>
```

Comment:
why in here we have a section and `AboutTextContainer` Is a section? Do we nest sections here? Is it the right approach? Just asking :)